### DISCUSSION TITLE
Subcircuit multiplicity

### DISCUSSION THREAD

**Discussion by eskiyerli** [Xyce/Xyce#35] [Q&A] [answered]
I have been trying to make Skywater process SPICE model libraries usable with Xyce. One issue that came up is the use multiplication factor, m. It was suggested that it is implicitly defined for subcircuits. Is that the case? As an example, can we just write:
Xsub node1 node2 ... SubcircuitName  Params: m=2

even if it is not defined as a parameter for the subcircuit?

**Comment by Karlsefni2012** [comment, marked as answer]
Unfortunately, Xyce doesn't yet support the `m=` multiplier for subcircuits. Xyce currently only supports it on the device level, in some devices.

**Comment by eskiyerli** [comment]
Thanks for your reply Eric. I also posted it to he Xyce discussion for
Skywater PDK Slack channel.

**Comment by tvrusso** [comment]
FWIW, subcircuit multiplicity was added for Xyce 7.5, which is now available on github but which has not yet been formally released in binary versions.
